diff '--color=auto' -Nur a/BUILD.bazel b/BUILD.bazel
--- a/BUILD.bazel	1970-01-01 08:00:00.000000000 +0800
+++ b/BUILD.bazel	2024-06-04 22:41:07.136340583 +0800
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 The Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Authors: liubang (it.liubang@gmail.com)
+
+cc_binary(
+    name = "lemon",
+    srcs = [
+        "tool/lemon.c",
+    ],
+    copts = [
+        "-std=c99",
+        "-O2",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "lempar",
+    srcs = [
+        "tool/lempar.c",
+    ],
+    visibility = ["//visibility:public"],
+)
