diff --color=auto -Nur a/BUILD.bazel b/BUILD.bazel
--- a/BUILD.bazel	1970-01-01 08:00:00
+++ b/BUILD.bazel	2024-06-04 20:43:30
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 The Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Authors: liubang (it.liubang@gmail.com)
+
+cc_library(
+    name = "nanobench",
+    srcs = ["src/test/app/nanobench.cpp"],
+    hdrs = ["src/include/nanobench.h"],
+    includes = ["src/include"],
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "nanobench_hdrs",
+    hdrs = ["src/include/nanobench.h"],
+    includes = ["src/include"],
+    visibility = ["//visibility:public"],
+)
